segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/ThrottlerCalculator.java

```diff
@@ -75,7 +75,7 @@
      * Maximum size (in number of operations) of the OperationLog, above which maximum throttling will be applied.
      */
     @VisibleForTesting
-    static final int OPERATION_LOG_MAX_SIZE = 1_000_000;
+    static final int OPERATION_LOG_MAX_SIZE = 50_000;
```

Comment:
Looks like you meant 5_000_000 .

Reply:
I mean 50000, because if we set to 5 million, it will be really hard to reach that big backlog to even see the throttling. I lower down to just 50000 to make it likely to repro. It is just for verification of whether the ciritical segments will ever be throttled or not.

Reply:
okay. got it.

Reply:
just make sure to revert so that we don't accidently keep it that way.